help/en/html/tools/fastclock/index.shtml

```diff
@@ -34,52 +34,83 @@
+      up Fast Clock...</a>
+      
+      
+      This gives you a window where you can:
 
       <ul>
-        <li>Select the timebase source to control (typically, this
-        is the internal computer clock)</li>
-        <li>Set the rate at which the clock runs</li>
         <li>Set the time</li>
         <li>Start and stop the clock</li>
+        <li>Set the rate at which the clock runs</li>
+        <li>Select the timebase source to control (typically, this
+        is the internal computer clock)</li>
       </ul>
 
       <h2>Fast Clock Display</h2>
 
       <p>Once you have created a timebase by configuring it in that
-      window, you can open clocks that will display on your screen.
-      Currently, JMRI has three types available:</p>
+      window, you can open clocks that will display on your screen,
+      all with an optional pause/run button.</p>
+      
+      
+      <p>All clocks can be resized by dragging the window edges</p>
+      
 
       <ul>
-        <li>Analog clock - the traditional circular clock with
-        hands. You can resize it by dragging the window edges.
-        There's also a pause/run button.</li>
-        <li>Nixie clock - a digital clock that looks like the
-        old-style Nixie tubes</li>
-        <li>Panel clock - you can also embed a clock in a panel
-        (instead of a separate window) from the panel editor.</li>
-        <li>Pragotron clock - flip clock used in net Czechoslovak State Railways &ndash; &#268;SD 
-            <span class="since">since 4.19.2</span></li>
+        <li>
+        <p>Tools > Clocks > 
+        Analog clock - The traditional circular clock with hands.</p>
+        <a href="images/analog-clock-513x567.png"><img 
+        src="images/analog-clock-513x567.png" width="245" height="271"
+        alt="Analog Clock"></a>
+        </li>
+        
+        <li><p>Tools > Clocks > LCD clock - a digital clock.</p>
+        <a href="images/lcd-clock-928x286.png"><img 
+        src="images/lcd-clock-928x286.png" width="879" height="271"
+        alt="LCD Clock"></a>
+        </li>
+        
+        <li><p>Tools > Clocks > Nixie clock - a digital clock that looks like the
+        old-style Nixie tubes</p>
+        <a href="images/nixie-clock-1085x380.jpg"><img 
+        src="images/nixie-clock-1085x380.jpg" width="774" height="271"
+        alt="Nixie Clock"></a>
+        </li>
+        
+        <li><p>Tools > Clocks > Pragotron clock - flip clock used in net Czechoslovak State Railways &ndash; &#268;SD 
+            <span class="since">since 4.19.2</span></p>
+            <a href="images/pragotron-clock-854x271.jpg"><img 
+        src="images/pragotron-clock-854x271.jpg" width="854" height="271"
+        alt="Nixie Clock"></a>
+        </li>
+            
+            
+        <li><p>Panel clock - you can also embed a clock in a panel
+        from the panel editor.<br>
+        Alalog clocks can be embedded into Layout Editor panels (instead of a separate window) via 
```

Comment:
Minor typo "alalog" (not a reason to hold the PR, just if you happen to be pushing an update)